include/dev/RedundantADC.hpp

```diff
@@ -0,0 +1,60 @@
+#pragma once
+
+#include <EVT/io/ADC.hpp>
+
+namespace IO = EVT::core::IO;
+
+namespace HIB::DEV {
+
+/**
+ * This class allows processing readings from redundant ADCs and checking for errors.
+ */
+class RedundantADC {
+public:
+    /**
+     * Enum class representing the status of RedundantADC processing.
+     */
+    enum Status {
+        /** No error. */
+        OK = 0,
+        /** The reading is off by more than 1 percent margin but less than the specified margin*/
```

Comment:
Single line comments should have a space at the end before closing the comment
```suggestion
        /** The reading is off by more than 1 percent margin but less than the specified margin */
```